Signing/node-express/src/app.ts

```diff
@@ -64,8 +62,7 @@ app.get('/order-signed', async (req: Request, res: Response) => {
 });
 
 app.get('/orders', async (req: Request, res: Response) => {
-  const orders = await getOrders();
-
+  const orders = await getOrders(100);
```

Comment:
the `querySignatureOrders` query takes an optional `status`, you can consider including that in the request and show OPEN/CLOSED orders in seperate UI elements

Reply:
Good idea. 
Probably makes sense to add pagination as well in this case (now the app only shows the last 10 orders).